frontend/src/scenes/urls.ts

```diff
@@ -157,8 +157,8 @@ export const urls = {
     cohorts: (): string => '/cohorts',
     experiment: (id: string | number): string => `/experiments/${id}`,
     experiments: (): string => '/experiments',
-    experimentsSavedMetrics: (): string => '/experiments/saved-metrics',
-    experimentsSavedMetric: (id: string | number): string => `/experiments/saved-metrics/${id}`,
+    experimentsSavedMetrics: (): string => '/experiments/shared-metrics',
```

Comment:
since the url is changing can we change the name of this function and the one below too?